### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by fehguy**
This PR addresses the following items from the https://github.com/OAI/OpenAPI-Specification/issues/589#issuecomment-250816162 comment, with minor corrections and a general review:

* Paths Object
* Path Item Object
* Operation Object
* External Documentation Object

### CODE REVIEW

**Comment by fehguy** [comment]
@OAI/tdc Please review and approve

File: versions/3.0.md [hunk 1]
```diff
@@ -345,14 +345,14 @@ Field Pattern | Type | Description
 #### <a name="pathsObject"></a>Paths Object
 
 Holds the relative paths to the individual endpoints.
-The path is appended to the [`basePath`](#oasBasePath) in order to construct the full URL.
-The Paths may be empty, due to [ACL constraints](#securityFiltering).
+The path is appended to the [`Server Object`](#serverObject) in order to construct
```
Comment (ePaul) [inline versions/3.0.md]: Hmm, not sure how one can append a path (string) to an object. Maybe you want something like this?

> The path is appended to the base path build from the server object.

File: versions/3.0.md [hunk 2]
```diff
@@ -345,14 +345,14 @@ Field Pattern | Type | Description
...
+the full URL.  The Paths may be empty, due to [ACL constraints](#securityFiltering).
 
 ##### Patterned Fields
 
 Field Pattern | Type | Description
 ---|:---:|---
-<a name="pathsPath"></a>/{path} | [Path Item Object](#pathItemObject) | A relative path to an individual endpoint. The field name MUST begin with a slash. The path is appended to the [`basePath`](#oasBasePath) in order to construct the full URL. [Path templating](#pathTemplating) is allowed.
+<a name="pathsPath"></a>/{path} | [Path Item Object](#pathItemObject) | A relative path to an individual endpoint. The field name MUST begin with a slash. The path is appended to the [`Server Object`](#serverObject) in order to construct the full URL. [Path templating](#pathTemplating) is allowed.
```
Comment (ePaul) [inline versions/3.0.md]: Same here.

File: versions/3.0.md [hunk 3]
```diff
@@ -418,9 +418,8 @@ Field Name | Type | Description
...
-<a name="pathItemBasePath"></a>basePath | `string` | The base path on which the API is served, which is relative to the [`host`](#pathItemHost). This optional value will override the top-level [basePath](#oasBasePath) if present. If it is not included, the API is served directly under the `host`. The value MUST start with a leading slash (`/`). The `basePath` does not support [path templating](#pathTemplating). 
-<a name="pathItemSchemes"></a>schemes | [`string`] | The transfer protocol of the API. Values MUST be from the list: `"http"`, `"https"`, `"ws"`, `"wss"`. This optional value will override the top-level [schemes](#oasSchemes) if present. If the `schemes` is not included, the default scheme to be used is the one used to access the OpenAPI definition itself.
+<a name="pathItemTrace"></a>trace | [Operation Object](#operationObject) | A definition of a TRACE operation on this path.
+<a name="pathItemServer"></a>[Server Object](#serverObject) | An alternative `server` object to service all operations in this path.
```
Comment (ePaul) [inline versions/3.0.md]: This field is missing a name?

File: versions/3.0.md [hunk 4]
```diff
@@ -418,9 +418,8 @@ Field Name | Type | Description
...
-<a name="pathItemBasePath"></a>basePath | `string` | The base path on which the API is served, which is relative to the [`host`](#pathItemHost). This optional value will override the top-level [basePath](#oasBasePath) if present. If it is not included, the API is served directly under the `host`. The value MUST start with a leading slash (`/`). The `basePath` does not support [path templating](#pathTemplating). 
-<a name="pathItemSchemes"></a>schemes | [`string`] | The transfer protocol of the API. Values MUST be from the list: `"http"`, `"https"`, `"ws"`, `"wss"`. This optional value will override the top-level [schemes](#oasSchemes) if present. If the `schemes` is not included, the default scheme to be used is the one used to access the OpenAPI definition itself.
+<a name="pathItemTrace"></a>trace | [Operation Object](#operationObject) | A definition of a TRACE operation on this path.
```
Comment (fehguy) [inline versions/3.0.md]: If `trace` has a fixed response, perhaps describing it here doesn't make sense, or we need to say that "if you use `trace`, you can't describe any response!?"

**Comment by fehguy** [comment]
May remove `trace` after further review.

**Pull request merged**